pkg/kubelet/userns/userns_manager.go

```diff
@@ -266,6 +266,11 @@ func (m *UsernsManager) Release(podUID types.UID) {
 	if !utilfeature.DefaultFeatureGate.Enabled(features.UserNamespacesSupport) {
 		return
 	}
+	// This should never happen, but if it doesn it can crash the kubelet. Let's just handle it
```

Comment:
if this should never happen, crashing the kubelet seems correct--it's a programmign error I'd expect testing to catch and users complaining a lot about to prevent us from putting the kubelet's state in a weird spot

Reply:
Makes sense. Removed this commit now!